tests/test_templatetags.py

```diff
@@ -18,5 +18,5 @@ def test_get_all(self):
     def test_get_filtered(self):
```

Comment:
@cruz-evan If you unify this with the way the above test is written (as in our fork), then you don't need to worry about these small formatting differences.   Probably it should use json.loads instead of eval, since this is the context wherein we hit the issue that necessitated this PR.